pkgs/tools/games/steamback/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, pkgs
+, python3
+, fetchPypi
+, python3Packages
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "steamback";
+  version = "0.3.6";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-hvMPSxIfwwQqo80JCpYhcbVY4kXs5jWtjjafVSMrw6o=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    pythonRelaxDepsHook
+  ];
+
+  buildInputs = with python3.pkgs; [
+    setuptools
+    pillow
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; with python3Packages; [
+    psutil
+    async-tkinter-loop
+    timeago
+    platformdirs
+    sv-ttk
+  ];
+
+  pythonRelaxDeps = [
+    "async-tkinter-loop"
+    "platformdirs"
+    "Pillow"
+  ];
+
+  makeWrapperArgs = [
+    ''--prefix TK_LIBRARY : "${pkgs.tk}/lib/tk8.6"''
+  ];
```

Comment:
```suggestion
```

Fixed two weeks ago.

Reply:
Ah excellent! This PR has been a while underway and I did not notice. Thanks! :)